src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ServiceAdminRoutes.scala

```diff
@@ -53,6 +54,13 @@ trait ServiceAdminRoutes extends SecurityDirectives with SamRequestContextDirect
               .map(users => (if (users.nonEmpty) OK else NotFound) -> users)
           }
         }
+      } ~
+      postWithTelemetry(samRequestContext) {
```

Comment:
shouldnt this be get?

Reply:
I'm thinking POST, since the number of IDs could get pretty unwieldy in a url.